pa11y_scripts/.pa11y-ci-p8-confirmation.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This is great, we should also add it to the checks we run at build time
https://github.com/usdoj-crt/crt-portal/blob/develop/.circleci/config.yml#L63